src/scanner.js

```diff
@@ -201,13 +201,16 @@ module.exports = function ({ config, db, logger }) {
         let dur = parseFloat(json.format.duration) || (1 / 24)
 
         let type = ' AUDIO '
-        if (json.streams[0].pix_fmt) {
-          type = dur <= (1 / 24) ? ' STILL ' : ' MOVIE '
 
-          const fr = String(json.streams[0].avg_frame_rate || json.streams[0].r_frame_rate || '').split('/')
-          if (fr.length === 2) {
-            tb = [ fr[1], fr[0] ]
-          }
+        for (let stream in json.streams) {
+          if (json.streams[stream].pix_fmt) {
+            type = dur <= (1 / 24) ? ' STILL ' : ' MOVIE '
+            const fr = String(json.streams[stream].avg_frame_rate || json.streams[stream].r_frame_rate || '').split('/')
+            if (fr.length === 2) {
+              tb = [ fr[1], fr[0] ]
+            }
+            break
+		  }
```

Comment:
Should be a `for of` loops since `streams` is an array?

Reply:
`for (const { pix_fmt } of json.streams) {`

Reply:
the problem with this is that it will set audio files with a cover as image instead of audio... so we also need to check the disposition of the stream

Reply:
Yes a good point, how about if we also check to see if there is a codec_type:audio element in one of the streams? This would have the same issue if the image stream is before the audio one (not sure if that can happen) but I would say that is a far less likely occurrence to audio being before video which seems to happen in around 30% of videos that I receive!

Reply:
I have spotted the deliberate mistake in my previous comment... We would also need some way to determine which is the "primary" stream I guess?

Reply:
I'm sorry, but isn't it more logical and practical than if there is video to switch to the video stream and take the data from it? I have problems when Vegas exports on Mainconcept mp4, stream 1 Audio, and 2 Video.